### ISSUE TITLE
Error while pip install

### ISSUE THREAD

**Issue by SpaceWhite** [shellphish/shellphish-afl#16]
Tried pip install this library on ubuntu 16.04

But I got this error

```
  I: Retrieving InRelease
  I: Failed to retrieve InRelease
  I: Retrieving Release
  I: Retrieving Release.gpg
  I: Checking Release signature
  gpgv: WARNING: multiple signatures detected.  Only the first will be checked.
  gpgv: Signature made Sat Jul  6 09:36:55 2019 UTC using RSA key ID 2B90D010
  gpgv: Good signature from "Debian Archive Automatic Signing Key (8/jessie) <ftpmaster@debian.org>"
  I: Valid Release signature (key id 126C0D24BD8A2942CC7DF8AC7638D0442B90D010)
  E: Invalid Release file, no entry for main/binary-mips/Packages
  error: Unable to fetch libraries
```

**Comment by bwuzhang** [comment]
@SpaceWhite I run into the same error. Have you found a fix for this? Thanks.

**Comment by bwuzhang** [comment]
I fixed it with the new pull request #17
